### ISSUE TITLE
Dockerize

### ISSUE THREAD

**Issue by bclindner** [bclindner/chipbot2#10]
Depending on what the server's running on, it might be advantageous to dockerize the bot. This would be totally trivial with how Docker works with Node projects.

**Comment by ViridianForge** [comment]
Yeah, the server isn't currently running Docker, but that said - may not hurt for me to get with the times. The current chipbot and discord bot are currently set up as system services - but I'm open to improvements.
